### ISSUE TITLE
Any option other than Heroku?

### ISSUE THREAD

**Issue by jacky780001** [ayastreb/bandwidth-hero#76]
I am getting messages from Heroku that they'd be ending free dynos on 11/28/2022, so in order to continue using bandwidth-hero, is there any other option than Heroku or hosting own server?

**Comment by ayastreb** [comment]
Yes, please refer to proxy README - https://github.com/ayastreb/bandwidth-hero-proxy - there's a couple of links in "Self-hosted" section down bellow. Proxy server is just a node.js app, you can run it at your own hosting wihtout problems

**Comment by kaovilai** [comment]
A k8s deployment yaml would probably be a cool addition

**Comment by SSL-ACTX** [comment]
I used Vercel to deploy bandwidth-hero, it works flawlessly.

**Comment by JuanJSAR** [comment]
Any tutorial on how to do it?

**Comment by SSL-ACTX** [comment]
@JuanJSAR 
Well, you can just fork bandwidth-hero-proxy repo and add a vercel.json (read about it in vercel), and import it to Vercel. That's it.
